### DISCUSSION TITLE
Overlapping transparent objects

### DISCUSSION THREAD

**Discussion by RocketPunk** [bnpr/Malt#589] [Q&A] [answered]
Hey everyone,
I'm pretty new to MALT so maybe that was answered before, but for now I haven't seen it. Here's my issue: I have a character for which I want to make anime style eyes, and I figured the best way to have the glint in the eyes would be having a transparent sphere around my actual eyes, with the eyes receiving a texture later on. The problem is, since my character wears glasses, now I can see through the eyes. How can I fix that? Should I go with multiple layers or is there a trick for single layer?
Have a great one
![image_2024-10-29_190516328](https://github.com/user-attachments/assets/b92324cf-06cc-49d0-bed0-6f5ab61dc3bc)

**Comment by pragma37** [comment, marked as answer]
Hi!
Malt renders transparent objects by layers, by default is 2 in Material Preview and 4 in Render,
but you can change the value in the `Default Render` node tree:
![imagen](https://github.com/user-attachments/assets/e56cd7b7-ea2b-43c8-aa69-62ce1ce39646)

**Comment by RocketPunk** [comment]
It worked! And here I thought there was a conflict between my 2 materials. Whew, what a relief. Thanks!
